contracts/sol6/reserves/IWeth.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.6.6;
+
+import "../IERC20.sol";
```

Comment:
wondering.
does this file belong in reserves folder.
can we include it maybe from open zeppelin?